input/fsh/modules/participant.fsh

```diff
@@ -94,3 +86,59 @@ Description: "Research oriented patient"
 * extension contains AgeAtLastVitalStatus named age-at-last-vital-status 0..1
 * extension[age-at-last-vital-status] ^short = "Age or date of last vital status"
 
+
+
+//  Person Module
+
+Logical: CdmPerson
+Id: SharedDataModelPerson
+Title: "Shared Data Model for Research Persons"
+Description: "The **Shared data model for Person**"
+* identifier 1..1 string "Unique Person identifier."
+* participant 1..1 reference "The participant we are describing"
+
+Extension: AccessPolicy
+Id: access-policy
+Title: "Access policy"
+Description: "Access Policy Extension"
+* insert SetContext(ResearchSubject)
+* value[x] only Reference
+* valueReference 1..1
+* valueReference only Reference(Consent)
```

Comment:
We actually have a [profile](https://nih-ncpi.github.io/ncpi-fhir-ig-2/StructureDefinition-ncpi-research-access-policy.html) for this that is a bit more consistent with our needs.